### PR TITLE
[DOC] Add fMRIPrep collection information to FAQ

### LINKED ISSUE #717: Add information about extracting echo-wise preprocessed data from fMRIPrep to FAQ
### Summary

Our current recommendation when using fMRIPrep + tedana (instead of just t2smap) is to extract echo-wise preprocessed data from the working directory to the output directory, then run tedana on those native-space preprocessed files, and then apply the saved transforms to the native-space denoised data.

### Next Steps

1. Add info about this recommendation to FAQ.
2. Link to example code to extract the semi-preprocessed data from the fMRIPrep working directory.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #717.

Changes proposed in this pull request:

- Add information about collecting fMRIPrep data (with gist) to FAQ.
- Link to new FAQ in other sections.

### CODE REVIEW

**Comment by tsalo** [comment]
I cancelled almost all of the jobs because this is a docs PR. Can restart later as long as the RTD test looks good.

**Comment by tsalo** [comment]
It looks great, but if anyone knows how to limit the embedded window to a certain number of lines (like how it is with permalinks in GitHub issues/PRs), that would be awesome.

image

**Review by eurunuela — DISMISSED** [review]
LGTM! Thank you @tsalo

**Comment by dowdlelt** [comment]
Maybe have something that points to the original issue, #717 so that when this closes it, and it (inevitably?) breaks, people could quickly reopen this.

**Comment by tsalo** [comment]
Thanks @dowdlelt. Added in b9209f0.

File: docs/faq.rst [hunk 1]
```diff
@@ -4,6 +4,38 @@ FAQ
...
+Unfortunately, fMRIPrep's working directory structure is not stable across versions,
+so writing code to grab the relevant files from the working directory is a bit of a moving target.
+Nevertheless, we have some code (thanks to Julio Peraza) that works for version 20.2.1.
+
+.. warning::
+    We will try to keep the following gist up-to-date, but there is no guarantee that it will work for a given version.
+    Use it with caution!
+
+    If you do find that the gist isn't working for an fMRIPrep version >= 20.2.1,
+    please comment on https://github.com/ME-ICA/tedana/issues/717 and we will take a look at the problem.
```
Comment (eurunuela) [inline docs/faq.rst:32]: Maybe you could make it a link?

**Review by eurunuela — DISMISSED** [review]

File: docs/faq.rst (hunk 1 above)
Comment (tsalo) [reply docs/faq.rst:32]: Like refer to it as "Issue #\717" and have that point to the URL that's in there currently?

Comment (eurunuela) [reply docs/faq.rst:32]: Yes. I think it would look better on the docs.

Comment (tsalo) [reply docs/faq.rst:32]: Will do. Thanks!

Comment (tsalo) [reply docs/faq.rst]: Should be good as of 28050e9.

**Review by eurunuela — APPROVED** [review]
Lovely, thank you @tsalo !

**Review by jbteves — APPROVED** [review]
Looks good to me, thanks @tsalo ! It looks particularly nice to have the gist embedded, awesome.

**Comment by AaronAngJW** [comment]
Hi there,

Is there any possibility of adding a couple more information as to how to properly use this collect_fmriprep.py function?
- When exactly do you call this process in the script? 
- What to expect if the function was ran successfully?
- How to then integrate the output of this function onto the actual tedana process.

It'd be really really nice if we can even get some sort of graphical guide that could aid this process.

**Pull request merged**